std-bits/base/src/main/java/org/enso/base/net/URITransformer.java

```diff
@@ -81,4 +80,27 @@ public static String encodeForQuery(String value) {
   public static String encode(String value) {
     return URLEncoder.encode(value, StandardCharsets.UTF_8).replace("+", "%20");
   }
+
+  public static String encodeQuery(String rawQueryString) {
+    var parts = rawQueryString.split("&");
+    StringBuilder sb = new StringBuilder();
+    for (int i = 0; i < parts.length; i++) {
+      if (i > 0) {
+        sb.append("&");
+      }
+      var indexOfEquals = parts[i].indexOf("=");
+      var keyValue =
+          indexOfEquals == -1
+              ? new String[] {parts[i]}
+              : new String[] {
+                parts[i].substring(0, indexOfEquals), parts[i].substring(indexOfEquals + 1)
+              };
+      sb.append(encodeForQuery(keyValue[0]));
+      if (keyValue.length > 1) {
+        sb.append("=");
+        sb.append(encodeForQuery(keyValue[1]));
```

Comment:
Will this mean that any following `=` signs will be encoded? I think the browsers leave the additional `=` signs unchanged so we may prefer that too.

Reply:
Tested in Postman and it seems to encode additional `=` signs.
![image](https://github.com/enso-org/enso/assets/4699705/349bc696-70c9-4b46-bcbc-f113efea92bb)

I have gone with that but if we find its problematic we can change our minds!